RossBlassingame/JPL-CUSeniorProjects
# Unified packages and fixed tests

What changes were made:
Unified org.geotools and mars packages, fixed tests to work on current class diagram, and removed the Route class.

What are the relevant JIRA tickets:
MARS-91

How can these changes be tested(test #?):
The updated tests can be verified by running the AppTest suite. Everything else is structural.

Comments:
Once this is merged, I should be ready to make a second pull request for the slope calculation work.
